tests/SegmentManagerTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace CachetHQ\Tests\Segment;
+
+use GrahamCampbell\TestBench\AbstractTestCase as AbstractTestBenchTestCase;
+use Mockery;
+use CachetHQ\Segment\SegmentManager;
+
+class SegmentManagerTest extends AbstractTestBenchTestCase
```

Comment:
You can't copy tests out of my existing packages without including the original license and a link back.